packages/docusaurus-plugin-openapi/src/sidebars/index.ts

```diff
@@ -5,15 +5,74 @@
  * LICENSE file in the root directory of this source tree.
  * ========================================================================== */
 
-import { ApiMetadata, ApiPageMetadata } from "../types";
+import path from "path";
+
+import { NavbarItem } from "@docusaurus/theme-common";
```

Comment:
If we use, we can include `@docusaurus/theme-common` as a dev dependency:
```suggestion
import type { NavbarItem } from "@docusaurus/theme-common";
```

Reply:
switched to PropSidebarItem from packages/docusaurus-plugin-openapi/src/plugin-openapi.d.ts